frontend/src/components/Board/Readme.js

```diff
@@ -22,7 +22,7 @@ const Readme = ({ renderedReadme }) => (
         max-width: 100%;
       }
 
-      #readme .highlight {
+      #readme pre {
```

Comment:
?

Reply:
I copied the `.highlight` from the original styles, this class doesn't exist in the readme rendered by Gitea.

Aside: this bug (the overflow) is reproducible on kitspace.org, see ulx3x board on a small screen.